### PR TITLE
Create Getting Started page

### PR DESCRIPTION
**Pull request by mazzystr**
This pull requests creates a Getting Started page in user-guide so that new users and contributors have a nice itinerary of tasks to complete to get up to speed with the project.

### CODE REVIEW

**Comment by codificat** [comment]
An initial comment about scope/goals: 

> ... so that new users and contributors have a nice itinerary of tasks to complete to get up to speed with the project.

You mention _users_ and contributors, but the content of the new page seems to be quite focused on contributors (e.g. the first line is about signing the CNCF CLA, which users certainly don't have to do)

**Comment by mazzystr** [comment]
> An initial comment about scope/goals:
> 
> > ... so that new users and contributors have a nice itinerary of tasks to complete to get up to speed with the project.
> 
> You mention _users_ and contributors, but the content of the new page seems to be quite focused on contributors.
> 
> Should this be a _Contributing_ page instead of _Getting Started_ ?

No idea...  We already have a Getting Started page [here](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md) and a Contributing page [here](https://github.com/kubevirt/kubevirt/blob/master/CONTRIBUTING.md).

File: appendix/contributing.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Contributing
+---------------------------
+
+Prerequisites
+---------------------------
+
+Reviewing the following will prepare you for contributing
+
+* You need to be comfortable with git, GitHub workflow of fork, branch, commit, open pull request, reviewing changes, and merge to work effectively in the KubeVirt community.
+* You need to be familiar with writing code in golang.  See the [golang tour](https://tour.golang.org/welcome/1) to familiarize yourself.
```
Comment (codificat) [inline appendix/contributing.md:10]: This is if you want to contribute code... there are other ways to contribute (e.g. documentation, blog posts, raising issues, helping triage/reproduce...).

Maybe worth expanding / clarifying along those lines. You do talk about _other ways to contribute_ below already... point there?

File: appendix/contributing.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Read the Developer [contribution page](https://github.com/kubevirt/kubevirt/blob/master/CONTRIBUTING.md) and the [getting started page](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md).
+* Try the Deploy KubeVirt on [minikube](https://kubevirt.io/quickstart_minikube/) or [kind](https://kubevirt.io/quickstart_kind/) quickstart lab.
+* Try the [Using KubeVirt](https://kubevirt.io/labs/kubernetes/lab1.html) lab.
+
+Your first contribution
+---------------------------
+
+The following will help you decide where to start
+
+* Check a repository issues list for issues that make good entry points.
```
Comment (codificat) [inline appendix/contributing.md:21]: Mention the `good-first-issue` label?

File: appendix/contributing.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Open a pull request using GitHub to existing documentation.
```
Comment (codificat) [inline appendix/contributing.md:22]: Maybe it's just me, but I don't really understand this sentence... "to existing documentation"?

**Review by codificat — CHANGES_REQUESTED** [review]
Some comments.

Besides that, the Welcome page has a section called _Developer_ which currently points to one of the links contained here (CONTRIBUTING.md). I believe that page should be updated to point to this new section instead.

**Comment by codificat** [comment]
This is still pending:

> Besides that, the Welcome page has a section called _Developer_ which currently points to one of the links contained here (CONTRIBUTING.md). I believe that page should be updated to point to this new section instead.

**Comment by mazzystr** [comment]
> at, the Welcome pag

To me the Welcome page is good enough.  (ref. [KNIECO-2537](https://issues.redhat.com/browse/KNIECO-2537)) We have a request to make a friendly page for incoming developers from IBM.  

Where do we want to go from here?  Update Welcome page with this content?  Update CONTIRBUTING page to point to the Welcome page?  Both those items feel out of scope for the story.

**Comment by codificat** [comment]
> Where do we want to go from here? Update Welcome page with this content? Update CONTIRBUTING page to point to the Welcome page? Both those items feel out of scope for the story.

I was just suggesting to replace the link in the welcome page that links directly to the CONTRIBUTING page with a link to this new content from this PR (which includes a link to CONTRIBUTING but provides more context).

**Comment by mazzystr** [comment]
@codificat Contributing link on Welcome page is updated

**Comment by codificat** [comment]
Ok, thanks.
/lgtm
/approve

**Review by codificat — APPROVED** [review]

**Pull request merged**